README.md

```diff
@@ -1,2 +1,4 @@
 # hello-world
-World, i just wanted to say hello. That's it.
+World, i just wanted to say hello.
+
+Well, I'm Loman, Willy Loman. I've come to Boston to make the biggest deal of my life.
```

Comment:
Isn't it a bit arrogant?
I propose something more subtle...
```suggestion
Well, I'm Loman, Willy Loman. I've come to Boston to find new chances for my business.
```

Reply:
Am I not talking to myself?
Willy, it's time to go back home.
And not... I don't accept these changes!!!